configurations/admin.template.ini

```diff
@@ -163,6 +163,14 @@ moduls.kalturaLiveStreamTranscode.basePermissionType = 2
 moduls.kalturaLiveStreamTranscode.basePermissionName = FEATURE_KALTURA_LIVE_STREAM
 moduls.kalturaLiveStreamTranscode.group = GROUP_ENABLE_DISABLE_FEATURES
 
+moduls.liveStreamSegmentDuration.enabled = true
```

Comment:
Done forget to add this change to the admin.ini file as well.
Mayne change "Enable Segment Duration Update" to "Dynamic segment duration support"